packages/runtime/container-runtime/src/connectionTelemetry.ts

```diff
@@ -63,10 +69,40 @@ class OpPerfTelemetry {
         this.deltaManager.on("disconnect", () => {
             this.opSendTimeForLatencyStatisticsForMsnStatistics = undefined;
             this.clientSequenceNumberForLatencyStatistics = undefined;
+            this.clientSequenceNumberForSittingInProcessingQueue = undefined;
             this.connectionOpSeqNumber = undefined;
             this.firstConnection = false;
         });
 
+        this.deltaManager.outbound.on("push", (messages) => {
+            for (const msg of messages) {
+                if (msg.type === MessageType.Operation && msg.clientSequenceNumber % numOpsToThrottle === 1) {
+                        this.clientSequenceNumberForSittingInProcessingQueue = msg.clientSequenceNumber;
```

Comment:
You should not overwrite clientSequenceNumberForSittingInProcessingQueue if it's already not undefined. While not very common, but if client sends a top of ops, then you will keep moving this.clientSequenceNumberForSittingInProcessingQueue forward before receiving acks, and thus may never report anything